android/src/main/java/com/getcapacitor/community/tts/TextToSpeech.java

```diff
@@ -39,12 +40,19 @@ public void onInit(int status) {
         this.initializationStatus = status;
     }
 
+
+
+
+    //A brief note on the voice property:
+    //voice is an index for the voice, and should probably be replaced by a string in the future (or perhaps with a voiceURI option in-lieu of voice)
+    //As is, we need to assign indices to the set in order to match things. 
```

Comment:
Please remove the comment as it does not add value. It is already documented that Voice is an index and instead of writing that it should be replaced by a string, it would be better to create an issue for it directly.